src/reliability.jl

```diff
@@ -0,0 +1,74 @@
+struct CronbachAlpha{T <: Real}
+    alpha::T
+    dropped::Vector{T}
+end
+
+function Base.show(io::IO, x::CronbachAlpha)
+    @printf(io, "Cronbach's alpha for all items: %.4f\n", x.alpha)
+    isempty(x.dropped) && return
+    println(io, "\nCronbach's alpha if an item is dropped:")
+    for (idx, val) in enumerate(x.dropped)
+        @printf(io, "item %i: %.4f\n", idx, val)
+    end
+end
+
+"""
+    cronbachalpha(covmatrix::AbstractMatrix{<:Real})
+
+Calculate Cronbach's alpha (1951) from a covariance matrix `covmatrix` according to
+the [formula](https://en.wikipedia.org/wiki/Cronbach%27s_alpha):
+
+```math
+\\rho = \\frac{k}{k-1} (1 - \\frac{\\sum^k_{i=1} \\sigma^2_i}{\\sigma^2_X})
+```
+
+where ``k`` is the number of items, i.e. columns; ``\\sigma_i`` denotes item variance;
+and ``\\sigma^2_i`` consists of item variances and inter-item covariances.
+
+Returns a `CronbachAlpha` object that holds:
+
+* `alpha`: the Cronbach's alpha score for all items, i.e. columns, in `covmatrix`; and
+* `dropped`: a vector giving Cronbach's alpha scores if a specific item,
+  i.e. column, is dropped from `covmatrix`.
+
+# Example
+```jldoctest
+julia> using StatsBase
+
+julia> cov_X = [10 6 6 6;
+                6 11 6 6;
+                6 6 12 6;
+                6 6 6 13];
+
+julia> cronbachalpha(cov_X)
+Cronbach's alpha for all items: 0.8136
+
+Cronbach's alpha if an item is dropped:
+item 1: 0.7500
+item 2: 0.7606
+item 3: 0.7714
+item 4: 0.7836
```

Comment:
To fix doctest failure:
```suggestion
item 4: 0.7826
```